main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"code-review-bot/internal"
+	"code-review-bot/internal/web"
+	"code-review-bot/internal/web/routes"
+	"context"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v68/github"
+	"github.com/gregjones/httpcache"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/openai/openai-go"
+	"github.com/openai/openai-go/option"
```

Comment:
Consider validating the GitHub and OpenAI tokens before initializing clients to prevent unnecessary calls to external services that could fail due to invalid tokens.